Userland/Utilities/tail.cpp

```diff
@@ -55,14 +59,18 @@ static ErrorOr<off_t> find_seek_pos(Core::File& file, int wanted_lines, bool sta
         return pos;
     }
 
+    // If we want the first or zeroeth line, we don't need to seek at all.
+    if (wanted_lines == 0 || wanted_lines == 1)
+        return 0;
```

Comment:
I might be missing something, but wouldn't `wanted_lines == 0` always be false here? It looks like we do the same check above that returns immediately if true, and since `wanted_lines` has not been modified when we perform this check again it should be false.

Reply:
Good catch, that needed to be inside the if case.